task/03-date-tasks.js

```diff
@@ -76,7 +79,19 @@ function isLeapYear(date) {
  *    Date(2000,1,1,10,0,0),  Date(2000,1,1,15,20,10,453)   => "05:20:10.453"
  */
 function timeSpanToString(startDate, endDate) {
-   throw new Error('Not implemented');
+
+    function xx (t, x) {
+        x = x || 2
+        return ('00'+t).slice(-x)
+    }
+
+    let ms = (endDate - startDate) % 1000;
+    let s = ((endDate - startDate) - ms) / 1000;
+    let secs = s % 60;
+    s = (s - secs) / 60;
+    let mins = s % 60;
+    let hrs =  (s - mins) / 60;
+    return xx(hrs) + ':' + xx(mins) + ':' + xx(secs) + '.' + xx(ms, 3);
```

Comment:
try to solve this task using Date methods
https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Global_Objects/Date